sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsRecordedTestSanitizer.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Core.Testing
+{
+    public class TextAnalyticsRecordedTestSanitizer : RecordedTestSanitizer
```

Comment:
LOL! I was about to push my changes to https://github.com/Azure/azure-sdk-for-net/pull/9523 to include recordings.
:p

Reply:
Do you have ARM template?